### PR TITLE
Last line v2

### PR DESCRIPTION
**Pull request by esharf**
## Status
In Progress

##Related Issues
https://github.com/demisto/etc/issues/20325

## Description
Last line v2 implementing:
file
lastline-upload-url
lastline-upload-file
lastline-get-report
lastline-chek-status
lastline-get-task-list

## Does it break backward compatibility?
   - No

## Must have
- [x] Tests
- [x] Documentation (with link to it)
- [x] Code Review

## Technical writer review
Mention and link to the files that require a technical writer review.

### CODE REVIEW

**Comment by bakatzir** [comment]
@esharf @yaakovi when finished, let's discuss how we are gonna test lastline, lastline-v2 with our current quota.

**Comment by ronykoz** [comment]
@yaakovi or @bakatzir can we finalise this review? 
I might need to use this in an upcoming POC
@reutshal FYI

**Comment by bakatzir** [comment]
@ronykoz @reutshal If yaakovi won't be able to review soon, you can still use the old JS version.
It's working.

**Comment by kirbles19** [comment]
@esharf Done reviewing integration and changelogs. @yaron-libman is reviewing the playbook and will add docs-approved when he's done.

**Comment by yaron-libman** [comment]
playbook review completed.

File: Integrations/Lastline_v2/Lastline_v2.py [hunk 1]
```diff
@@ -0,0 +1,281 @@
...
+    def get_task_list(self):
+        for param in ('before', 'after'):
+            if param in self.command_params:
+                self.command_params[param] = self.command_params[param].replace('T', ' ')
+        result = self.http_request('/analysis/get_completed')
+        if 'data' in result:
+            context_entry = self.get_status_and_time(argToList(result['data'].get('tasks')))
+            human_readable = '|UUID|Time|Status|\n|--|--|--|'
+            for uuid, task_time, status in context_entry:
+                human_readable += f'\n|{uuid}|{task_time}|{status}|'
```
Comment (yaakovi) [inline Integrations/Lastline_v2/Lastline_v2.py]: use tableToMarkdown

**Pull request merged**